lib/descriptor.js

```diff
@@ -27,61 +27,65 @@ const generateDescriptor = async (data, dpName = "scratchpad") => {
 }
 module.exports.generateDescriptor = generateDescriptor
 
-const generateDescriptorForLocalFiles = function (dpName = "scratchpad") { 
+const generateDescriptorForLocalFiles = (dpName = "scratchpad") => { 
   return new Promise(function(resolve, reject) { 
     fs.readdir('./', (err, files) => {
       var descriptor = {
```

Comment:
@Mikanebu I did not commented on every single line where `var` is used instead of `let`. Go through the file and replace all of them